vcbuild.bat

```diff
@@ -14,7 +14,7 @@ if /i "%1"=="/?" goto help
 @rem Process arguments.
 set config=Release
 set target=Build
-set target_arch=x86
+set target_arch=x64
```

Comment:
https://github.com/nodejs/node/pull/10156 was `semver-major`, so I guess this one is too. I'd be fine with that though since most people use `vcbuild` to build master anyway.